docker-images/manylinux2014-x64-ruby/Dockerfile

```diff
@@ -0,0 +1,52 @@
+# See: https://github.com/pypa/manylinux
+# and: https://github.com/pypa/python-manylinux-demo
+FROM quay.io/pypa/manylinux2014_x86_64
+
+###############################################################################
+# Basics
+###############################################################################
+RUN yum -y install sudo cmake3 \
+    && yum clean all \
+    && ln -s `which cmake3` /usr/bin/cmake \
+    && ln -s `which ctest3` /usr/bin/ctest \
+    && cmake --version \
+    && ctest --version
+
+###############################################################################
+# Python/AWS CLI
+###############################################################################
+RUN /opt/python/cp37-cp37m/bin/python -m pip install --upgrade pip setuptools virtualenv \
+    && /opt/python/cp37-cp37m/bin/python -m pip install --upgrade awscli \
+    && ln -s `find /opt -name aws` /usr/local/bin/aws \
+    && which aws \
+    && aws --version
+
+###############################################################################
+# Install pre-built CMake
+###############################################################################
+WORKDIR /tmp
+RUN curl -sSL https://d19elf31gohf1l.cloudfront.net/_binaries/cmake/cmake-3.13-manylinux1-x64.tar.gz -o cmake.tar.gz \
```

Comment:
it looks like you're putting cmake on the machine twice? once through yum (lines 8-13) and again here

Reply:
I copied this from the[ aws-crt-builder Dockerfile](https://github.com/awslabs/aws-crt-builder/blob/main/.github/docker-images/manylinux2014-x64/Dockerfile).  Not sure if its required or not, but didn't want to change it...  

Also I don't have permissions to any of the aws-crt-builder images (to even read) which is why I duplicated the Dockerfile rather than simpying using a `FROM` with the crt manylinux2014-x86 image.  If you know how to get those permissions working, that would probably be the ideal setup.